core/types/crosslink.go

```diff
@@ -23,6 +24,16 @@ type CrossLink struct {
 	EpochF       *big.Int
 }
 
+// RPCCrossLink is used for returning data to RPC calls, converting byte arrays to hex encoded strings
+type RPCCrossLink struct {
```

Comment:
Not needed to make this data type wrapper, instead just define a marshaljson method on the crosslink type itself , return a crosslink type in rpc

Reply:
Made the struct, so Signature & SignatureBitmaps would return something as a hex encoded string, instead of a byte array, to match latestHeader.

Reply:
I know, you can control that in marshaljson method of cross link since the marshaljson method controls presentation